src/libraries/System.Drawing.Common/src/System/Drawing/Imaging/CachedBitmap.Unix.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.CodeAnalysis;
+using Gdip = System.Drawing.SafeNativeMethods.Gdip;
+
+namespace System.Drawing.Imaging
+{
+    public sealed partial class CachedBitmap
+    {
+        internal static bool IsCachedBitmapSupported()
```

Comment:
Maybe the result should be cached?

Edit: Ah, nevermind. I see the use in `CachedBitmap`, and there it should be cached.